### PR TITLE
Fix django migrations while maintaining DBViews

### PR DESCRIPTION
**Pull request by jackleland**
This is my fix for the problem with migrations caused by adding the group profile into the un-managed `atlas` app, basically involving refactoring the changes out into a separate app called `accounts` which is managed by django and leaves the `atlas` folder unbothered by the django `makemigrations` command. 

Along the way I found a problem with the previous commit anyway whereby the signal which attempted to auto-create the GroupProfile of any created Group was at odds with Groups created through the admin interface, and therefore not allowing the creation of groups. I ended up removing the signal as there is not likely to be group creation happening anywhere other than through the admin interface anyhow, though we can discuss this obviously.

Also note that I have included the migrations for this new accounts app, which is standard practice for django (and not a problem for production given that this is a new app and doesn't have historical, uncommitted migrations) but **let me know thoughts** on that too.

### CODE REVIEW

**Comment by jackleland** [comment]
@genghisken @HeloiseS

File: psat_server_web/atlas/accounts/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Generated by Django 4.1.13 on 2024-10-29 11:23
```
Comment (genghisken) [inline psat_server_web/atlas/accounts/migrations/0001_initial.py:1]: This file should not be in the pull request. Can we remove it and the entire migrations directory. No migrations should ever be checked in. I suspect this was added by accident!

**Review by genghisken — CHANGES_REQUESTED** [review]
Can you remove the atlas/atlas/migrations/ directory and its contents from the PR? I believe this was checked in by accident. Thanks!

File: psat_server_web/atlas/accounts/migrations/0001_initial.py (hunk 1 above)
Comment (jackleland) [reply psat_server_web/atlas/accounts/migrations/0001_initial.py:1]: Happy to remove but not done by accident, committing migrations allows the whole database to be reproduced in a dev environment easily and (I think) is standard practice. Django wrote them to be version controlled and sometimes we will have to write manual migrations, which will definitely need to be checked in! We can discuss this afternoon of course :)

**Comment by jackleland** [comment]
Migrations removed, can discuss later :)

**Pull request merged**